taxonomies/categories.txt

```diff
@@ -40455,6 +40656,12 @@ nl:Varkensvleesmaaltijden
 pt:Refeiçoes com porco
 zh:猪肉餐
 
+<en:Pork meals
```

Comment:
should probably be "pork meat" and not "pork meals"

Reply:
resolu